src/lib/services/identity.ts

```diff
@@ -17,6 +52,16 @@ const getMemberByAlias = async (alias: string) => {
   throw new HttpResponse({})
 }
 
+const getHealth = async () => {
+  const res = await fetch(`http://${env.IDENTITY_SERVICE_HOST}:${env.IDENTITY_SERVICE_PORT}/health`)
```

Comment:
I'd recommend refactoring the way `URL_PREFIX` is constructed to depuplicate logic here. At some point we'll likely enable https and it would be nice to only need to change this in one place